build/rules/js/js.build_defs

```diff
@@ -27,81 +26,69 @@ def pnpm_install(
         srcs = [fg],
         exported_deps = [fg],
         output_is_complete = False,
-        building_description = "Installing pnpm dependencies...",
-        outs = ["pnpm.state"],
+        building_description = "Installing Yarn dependencies...",
+        outs = ["yarn.state"],
```

Comment:
Yarn doesn't produce `yarn.state` file after doing install. It creates a `.yarn-integrity` file if I remember correctly, or something along these lines.